cfct/manifest.yaml

```diff
@@ -0,0 +1,121 @@
+# This is a sample, non-production-ready file
+# © 2021 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.  
+# This AWS Content is provided subject to the terms of the AWS Customer Agreement available at  
+# http://aws.amazon.com/agreement or other written agreement between Customer and either
+# Amazon Web Services, Inc. or Amazon Web Services EMEA SARL or both.
+
+---
+#Default region for deploying Custom Control Tower: Code Pipeline, Step functions, Lambda, SSM parameters, and StackSets
+region: ap-southeast-2
+version: 2021-03-15
+
+# Control Tower Custom CloudFormation Resources
+resources: 
+
+  - name: VPCFlowLogsBucket
+    resource_file: templates/vpc_flowlog_logging_bucket.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "SSEAlgorithm"
+        parameter_value: "AES256"
+
+      - parameter_key: "KMSMasterKeyID"
+        parameter_value: ""
+    deploy_method: stack_set
+    deployment_targets:
+      accounts:
+        - Audit
+    export_outputs:
+      - name: /org/sharedservice/networking/vpcflowlogsbucket
+        value: $[output_oBucketName]
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogsMgmtRole
+    resource_file: templates/vpc_flowlog_mgmt.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "EventBusDestinationAccount" # Typiclly this will be the NetworkHub account
+        parameter_value: "<REPLACE ME>"
+
+      - parameter_key: "BaselineCloudTrailStackArn" # Copy this this from mgmt account in cloudformation stacksets for AWSControlTowerBP-BASELINE-CLOUDTRAIL
+        parameter_value: "<REPLACE ME>"
+    deploy_method: stack_set
+    deployment_targets:
+      accounts:
+        - Management
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogRoleInSpoke
+    resource_file: templates/vpc_flowlog_spoke.template
+    parameters:
+      - parameter_key: "OrganizationId"
```

Comment:
this param does not exist in the template
error : An error occurred (ValidationError) when calling the CreateStackSet operation: Parameters: [ControlTowerMasterRegion, OrganizationId] do not exist in the template

Reply:
actually ignore this - the real cause is due to missing param `OrganizationId` on the template itself.
this param is required in order to create logical resource `FlowLogHubAssumeRole` which also missing from the `vpc_flowlog_spoke.template`  

Ref to [FlowLogHubAssumeRole](https://github.com/aws-samples/aws-ct-vpc-flowlogs/blob/main/template/ct_vpc_flowlog_stackset.yml#L379)

Reply:
yep I missed out my latest change which had the role in the spoke template.. latest commit will fix this